code/presets/create-react-app/src/index.ts

```diff
@@ -78,12 +78,11 @@ const webpack = async (
 
   // Remove existing rules related to JavaScript and TypeScript.
   logger.info(`=> Removing existing JavaScript and TypeScript rules.`);
-  const filteredRules =
-    webpackConfig.module &&
-    webpackConfig.module.rules.filter(
-      ({ test }: RuleSetRule) =>
-        !(test instanceof RegExp && ((test && test.test('.js')) || test.test('.ts')))
-    );
+  const filteredRules = webpackConfig.module?.rules?.filter((rule: RuleSetRule | '...') => {
```

Comment:
I don't understand this. Where does the string literal '...' come from?

Reply:
Its defined type of rule in webpack types `code\node_modules\webpack\types.d.ts`.

```
	/**
	 * An array of rules applied for modules.
	 */
	rules?: (RuleSetRule | "...")[];
```

I didn't figure any way how to avoid it.